locales-en-US.xml

```diff
@@ -313,6 +550,19 @@
     <term name="editortranslator" form="verb">edited &amp; translated by</term>
 
     <!-- SHORT VERB ROLE FORMS -->
+    <term name="chair" form="verb-short"></term> <!-- ??? -->
+    <term name="compiler" form="verb-short">compiled by</term>
+    <term name="contributor" form="verb-short">contributions by</term>
```

Comment:
@adam3smith @rmzelle @fbennett Can you remind me, if a form is included here, but blank, that is treated as "missing" and falls back to the long form, correct? So many of these could be reduced to blank if the default should be fallback?

```suggestion
    <term name="contributor" form="verb-short">with</term>
```

Reply:
No, it only falls back to long when the term is missing, not when it's blank. 
A blank term means nothing is being rendered (as you do for the timestamp); that extends to short forms as well (just tested with citeproc-js; not sure if the behavior is in the specs -- I don't think so -- but I think it's correct).

Reply:
@adam3smith Thanks. In that case, could you help out with the short forms for the verbs here as well? I'm not sure how they would generally be abbreviated.